pkg/manifests/bindata.go

```diff
@@ -946,11 +946,13 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//     data/
-//       foo.txt
-//       img/
-//         a.png
-//         b.png
+//
```

Comment:
This is why ci/prow/verify is failing. It was spaces and now is indents. Is this a go version issue as you mentioned here: https://github.com/openshift/cluster-ingress-operator/pull/869#issuecomment-1351768652?

Reply:
Sorry!  Fixed in https://github.com/openshift/cluster-ingress-operator/compare/ba9df6ba6efecdb5b40eabb06cb9dfe20df36d75..be7ae90c7363152360fc01a913def9c21d49dc62.